scripts/updateBuildPlugins.sh

```diff
@@ -0,0 +1,8 @@
+CMAKE_BINARY_DIR=../build/
+> $CMAKE_BINARY_DIR/lib/Camera.plugin
+for i in find ../isis/src/*/objs/*/Camera.plugin
+  do
+    if test -f "$i"; then
+        cat "$i" >> $CMAKE_BINARY_DIR/lib/Camera.plugin
```

Comment:
This looks like it will concatenate to the lib/Camera.plugin file every time it runs. May want to remove the existing file before the loop. I'm reasonable sure the plugin handling routines only search for the first occurrence in the Camera.plugin file, so new plugin entries added to the bottom of the file will not be used.